Makefile

```diff
@@ -153,18 +153,16 @@ buildconfig:
 	git checkout -- kolibri/utils/build_config # restore __init__.py
 	python build_tools/customize_build.py
 
-dist: writeversion staticdeps staticdeps-cext buildconfig assets translation-django-compilemessages
+dist: writeversion staticdeps staticdeps-cext buildconfig translation-extract assets translation-django-compilemessages
```

Comment:
@rtibbles I need to ask about this a bit late: But why is `translation-extract` added here? When building a dist, it would seem unexpected if suddenly the .po files etc are changed... I will happily revert it in #4056 since that's where it conflicts :)

Reply:
We need a copy of the en frontend message files in the build, and they no longer get generated by make assets. If you can modify it to just extract frontend messages, that would be great.